src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/CleanupUnusedObjectsInCurrentContextTests.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public class CleanupUnusedObjectsInCurrentContextTests
+    {
+        [Fact]
+        public void CleanupUnusedObjectsInCurrentContext_Invoke_Success()
```

Comment:
I am missing something here. The function is Invoke -> Success. Why is the function being invoked twice? If there is a reason, please comment so when it fails we know why.

Reply:
Updated the function name and comment